lib/cisco_node_utils/vxlan_vtep.rb

```diff
@@ -135,6 +135,33 @@ def default_source_interface
       config_get_default('vxlan_vtep', 'source_intf')
     end
 
+    def source_interface_hold_down_time
+      config_get('vxlan_vtep', 'source_intf_hold_down_time', name: @name)
+    end
+
+    def source_interface_hold_down_time=(time)
+      # Source interface should be configured before hold down time
+      # is configured.
+      return if source_interface == default_source_interface
```

Comment:
What happens if you try to set the hold-down-time in the CLI without setting the interface? Does it generate a meaningful error message?